BuildResidentialHPXML/measure.rb

```diff
@@ -1069,6 +1083,12 @@ def arguments(model)
     arg.setUnits('W/CFM')
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('cooling_system_is_shared_system', true)
+    arg.setDisplayName('Cooling System: Is Shared System')
+    arg.setDescription('Whether the cooling system is a shared system. If true, assumed to serve all the units in the building. This is valid only for a chiller.')
+    arg.setDefaultValue(false)
+    args << arg
+
```

Comment:
This argument is not needed. IsSharedSystem in the HPXML file should always be true for chillers and always false for other cooling system types.